spanner/tests/system/test_system.py

```diff
@@ -824,11 +824,11 @@ def test_read_w_manual_consume(self):
         retrieved = 0
         while True:
             try:
-                streamed.consume_next()
+                streamed._consume_next()
```

Comment:
I'd argue that we don't want to exercise private methods in a system test.  This test should either be deleted, or else updated to use the expected "public" pattern.